src/components/editor/ResourceProperty.jsx

```diff
@@ -12,6 +12,9 @@ const _ = require('lodash')
 export class ResourceProperty extends Component {
   constructor(props) {
     super(props)
+    this.state = {
+      jsx: []
```

Comment:
Where is this state variable used? Cruft?

Reply:
Not sure why this variable is being set in state (this wasn't part of the my PR but will remove it if you want)

Reply:
Huh, that's weird. GitHub seems to think this is a change in your branch (vs. what's in `master`). Yeah, :+1: 	to removing. Thanks for offering!

Reply:
Done, removed in latest commit.